### ISSUE TITLE
Remember selected product variation on product page (frontend)

### ISSUE THREAD

**Issue by extrapixel** [woothemes/woocommerce#8501]
To keep the selected variation after reloads. For a better user experience when 3rd party plugins reload our product page (language-switching, currency-switching, etc).

**Comment by mikejolley** [comment]
This is already done in 2.4

**Comment by extrapixel** [comment]
This is still an issue. Reloading the page (e.g. via a currency switcher) does switch back to the default variation, no matter the selection before the reload.

**Comment by mikejolley** [comment]
@extrapixel Selections are not sticky - only after posting the form.
